Parse-Dashboard/parse-dashboard-config.json

```diff
@@ -1,10 +1,16 @@
 {
-  "apps": [{
-    "serverURL": "",
-    "appId": "",
-    "masterKey": "",
-    "appName": "",
-    "iconName": ""
-  }],
-  "iconsFolder": "icons"
+  "apps": [
+    {
+        "serverURL": "https://comerate2016.herokuapp.com/parse", 
+        "appId": "2016comrate",
+        "masterKey": "2016comrate1",
```

Comment:
You can probably close that PR and recycle your masterKey... No way to remove that now.